### ISSUE TITLE
segfault when running `lean -r`

### ISSUE THREAD

**Issue by Julian** [leanprover-community/lean#465]
### Prerequisites

* [x] Put an X between the brackets on this line if you have done all of the following:
    * Checked that your issue isn't already [filed](https://github.com/leanprover-community/lean/issues).
        * Specifically, check out the [wishlist](https://github.com/leanprover-community/lean/issues?q=is%3Aissue+is%3Aopen+label%3AI-wishlist), open [RFCs](https://github.com/leanprover-community/lean/issues?q=is%3Aissue+is%3Aopen+label%3ARFC),
        or [feature requests](https://github.com/leanprover-community/lean/issues?q=is%3Aissue+is%3Aopen+label%3AFeature).
    * Reduced the issue to a self-contained, reproducible test case.

### Description

Running `lean -r` produces a segfault:

```
⊙  lean -r                                                                                                                                                                                        julian@Air
zsh: segmentation fault  lean -r
```

I assume this isn't really too important and `-r` isn't actually an argument `lean` takes, but I discovered this trying to use `lean --doc` and mistakenly just copying `lean --doc=file -r` exactly as shown in the usage message, but the `--doc` doesn't seem required. Not sure exactly what the intention of that usage line is.

### Versions

```
⊙  lean --version                                                                                                                                                                                 julian@Air
Lean (version 3.20.0, commit 4c40c41ccc08, Release)
```

```
⊙  uname -a                                                                                                                                                                                       julian@Air
Darwin Air.lan 19.6.0 Darwin Kernel Version 19.6.0: Thu Jun 18 20:49:00 PDT 2020; root:xnu-6153.141.1~1/RELEASE_X86_64 x86_64 i386 MacBookAir6,2 Darwin
```

**Comment by gebner** [comment]
Oh, `--doc` is broken.  Maybe we should just remove it.  The mathlib API docs are generated using this other tool: https://github.com/leanprover-community/doc-gen/

**Comment by rwbarton** [comment]
Yeah, we could just remove the option handling code around `--doc` even. It's been broken since I started using Lean, and the only reason I know that is because it's listed in the `lean --help` output.
